### PR TITLE
implemented the hashCode and equals methods

### PR DESCRIPTION
**Pull request by JJ-Han**
Overrode the hashCode and equals methods in both the Event and Task classes

### CODE REVIEW

**Review by jolateral — COMMENTED** [review]
can you add javadocs please? :)

File: src/entity/Event.java
Comment (jolateral) [inline src/entity/Event.java:1]: on line 200, is it supposed to be `endDate.equals(event.endDate)` instead of `startTime.equals(endTime)`?

**Review by jolateral — COMMENTED** [review]
I left a comment on the event file!

File: src/entity/Event.java
Comment (JJ-Han) [reply src/entity/Event.java:1]: fixed a bug

**Comment by JJ-Han** [comment]
> can you add javadocs please? :)

Javadoc is now completed

**Review by jolateral — APPROVED** [review]
Looks great! Thanks for adding the javadocs and fixing the small error :) Approved.

**Pull request merged**